install.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+#set -ueo pipefail
+#set -x
+
+REPO_DIR=$(cd $(dirname $0) && pwd)
+
+DEST_DIR=/usr/share/icons
+THEME_NAME=La-Capitaine
+THEME_DIR=${DEST_DIR}/${THEME_NAME}
+
+
+echo  "Customize '${THEME_DIR}'..."
+  ./configure
+
+usage() {
+  printf "%s\n" "Usage: $0 [OPTIONS...]"
+  printf "\n%s\n" "OPTIONS:"
+  printf "  %-25s%s\n" "-d, --dest DIR" "Specify theme destination directory (Default: ${DEST_DIR})"
+  printf "  %-25s%s\n" "-n, --name NAME" "Specify theme name (Default: ${THEME_NAME})"
+  printf "  %-25s%s\n" "-h, --help" "Show this help"
+  printf "\n%s\n" "INSTALLATION EXAMPLES:"
+  printf "%s\n" "Install all theme variants into ~/.icons"
+  printf "  %s\n" "$0 --dest ~/.icons"
+}
+
+install() {
+  local dest=${1}
+  local name=${2}
+
+  local THEME_DIR=${dest}/${name}
+
+  [[ -d ${THEME_DIR} ]] && rm -rf ${THEME_DIR}
+
+  echo "Installing '${THEME_DIR}'..."
+
+  mkdir -p  ${THEME_DIR}
+  cp -r ${REPO_DIR}/configure ${THEME_DIR}
+  cp -r ${REPO_DIR}/COPYING ${THEME_DIR}
+  cp -r ${REPO_DIR}/index.theme ${THEME_DIR}
+  cp -r ${REPO_DIR}/LICENSE ${THEME_DIR}
+  cp -r ${REPO_DIR}/actions ${THEME_DIR}/actions
+  cp -r ${REPO_DIR}/animations  ${THEME_DIR}/animations
+  cp -r ${REPO_DIR}/apps  ${THEME_DIR}/apps
+  cp -r ${REPO_DIR}/devices ${THEME_DIR}/devices
+  cp -r ${REPO_DIR}/emblems ${THEME_DIR}/emblems
+  cp -r ${REPO_DIR}/emotes  ${THEME_DIR}/emotes
+  cp -r ${REPO_DIR}/mimetypes ${THEME_DIR}/mimetypes
+  cp -r ${REPO_DIR}/panel ${THEME_DIR}/panel
+  cp -r ${REPO_DIR}/places  ${THEME_DIR}/places
+  cp -r ${REPO_DIR}/status  ${THEME_DIR}/status
+  case `lsb_release -cs` in
```

Comment:
This "install.sh" is not platform agnostic. This is explicitly checking for Ubuntu release names. This project aims to be distro agnostic, so this needs to be made more generic if it is to be accepted.